src/components/QuoteCard/index.tsx

```diff
@@ -1,89 +1,63 @@
-import { Button } from "../Button";
+import { Quote } from "../..//types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp as faThumbsUpSolid } from "@fortawesome/free-solid-svg-icons";
+import { faThumbsUp as faThumbsUpRegular } from "@fortawesome/free-regular-svg-icons";
+import { faThumbsDown as faThumbsDownRegular } from "@fortawesome/free-regular-svg-icons";
 import {
-  useQuotesContext,
-  useQuotesDispatchContext,
+  useLikedQuotesContext,
+  useLikedQuotesDispatchContext,
 } from "../../QuotesContextProvider";
-import {
-  useQuoteIndexContext,
-  useQuoteIndexDispatchContext,
-} from "../../QuoteIndexContextProvider";
-import { useState } from "react";
-import { Quote } from "../../types";
+import { useAuth } from "../../AuthContextProvider";
 
-export const QuoteCard = ({ quote, author, likeCount } : Quote) => {
-  const quotes = useQuotesContext();
-  const currentIndex = useQuoteIndexContext();
-  const setQuotes = useQuotesDispatchContext();
-  const dispatchQuoteIndex = useQuoteIndexDispatchContext() || ((index: number) => {});
+interface QuoteCardProps extends Quote {
+  onProfilePage?: boolean;
+  isUserCreated?: boolean;
+  handleDeleteCreatedQuote?: (quote: Quote) => void;
+}
 
-  const [liked, setLiked] = useState(false);
-  const [isBouncing, setIsBouncing] = useState(false);
+export const QuoteCard = ({ quote, author, onProfilePage, isUserCreated, handleDeleteCreatedQuote }: QuoteCardProps) => {
+  const likedQuotes = useLikedQuotesContext() ?? [];
+  const { handleLike, handleDislike } = useLikedQuotesDispatchContext() ?? { handleLike: () => {}, handleDislike: () => {} };
```

Comment:
Something is not right here :) Update the custom hooks you have in the QuotesContextProvider to be as in the AuthContextProvider (useAuth). This way the code will be much cleaner.